src/assets/locales/fr.json

```
@@ -0,0 +1,567 @@
+  "enter-on-device-modal.short-url-directions": "Dans le navigateur web de votre appareil, aller à:",
+  "enter-on-device-modal.title": "Entrer depuis l'appareil",
+  "enter-on-device-modal.unsupported-browser": "WebVR n'est pas supporté sur ce navigateur, pour entrer avec Oculus ou SteamVR, utiliser Firefox.",
+  "exited-room-screen.closed-room-tos": "Une salle peut être fermée par le propriétaire de la salle, ou si nous recevons des signalements qui violent nos <toslink>Conditions d'utilisation</toslink>.",
+  "exited-room-screen.connect-tcp": "Vous pouvez essayer de <a>vous connecter via TCP</a>, qui peut mieux fonctionner sur certains réseaux.",
+  "exited-room-screen.contact-us": "Si vous avez des questions, contactez nous par {contactEmail}.",
+  "exited-room-screen.create-room": "Vous pouvez aussi <a>créer une nouvelle salle</a>.",
+  "exited-room-screen.no-longer-availible": "Désolé, cette salle n'est plus disponible.",
+  "exited-room-screen.reason.closed": "Cette salle n'est plus disponible.",
+  "exited-room-screen.reason.connect-error": "Impossible de se connecter à cette salle, s'il vous plait, essayez plus tard.",
+  "exited-room-screen.reason.denied": "Vous n'êtes pas autorisé(e) à rejoindre cette salle. S'il vous plait, demaner une permission au créateur de cette room.",
+  "exited-room-screen.reason.disconnected": "Vous avez été déconnecté(e) de la salle. Rafraichissez la page pour essayer de vous reconnecter.",
+  "exited-room-screen.reason.exited": "Votre session s'est terminée. Rafraichissez votre page de navigateur pour en démarrer une nouvelle.",
+  "exited-room-screen.reason.full": "La salle est pleine, s'il vous plait, essayez plus tard.",
+  "exited-room-screen.reason.left": "Vous avez quitté la salle.",
+  "exited-room-screen.reason.scene-error": "Le chargement de la scène a échoué.",
+  "exited-room-screen.reason.version-mismatch": "La version que vous avez déployé n'est pas encore disponible. Votre navigateur va se rafraichir dans 5 secondes.",
+  "exited-room-screen.source-link": "Si vous souhaitez utiliser votre propre serveur, le code source de Hubs est disponible sur <a>GitHub</a>.",
+  "footer.logo-alt": "Logo",
+  "footer.powered-by": "Propulsé par <a>Hubs Cloud</a>",
+  "footer.privacy-notice": "Notes sur la vie privée",
+  "footer.terms-of-use": "Conditions d'utilisation",
+  "footer.whats-new": "Quoi de neuf",
+  "header.admin": "Admin",
+  "header.cloud": "Hubs Cloud",
+  "header.community": "Communauté",
+  "header.docs": "Docs",
+  "header.sign-in": "Se connecter",
+  "header.sign-out": "Se déconnecter",
+  "header.signed-in-as": "Connecté en tant que {email}",
+  "header.source": "Source",
+  "home-page.add-to-discord": "Ajouter le bot de {appName} à Discord",
+  "home-page.communicate-blurb": "choisissez un avatar qui vous représente, mettez votre casque audio, et franchissez le pas. Hubs rend facile la connection avec les autres personnes dans votre salle privée par la voix, le chat par texte.",
```

```suggestion
  "home-page.communicate-blurb": "Choisissez un avatar qui vous représente, mettez votre casque audio, et franchissez le pas. Hubs facilite la communication entre les personnes, autant par la voix que par message texte.",
  Obviously none of us are professional translators 😅
```

---

```suggestion
  "home-page.communicate-blurb": "Choisissez un avatar qui vous représente, mettez votre casque audio et c'est parti ! Hubs vous permet de communiquer facilement les uns avec les autres, par la voix ou par message.",
```
I try to translate the whole thing without following the english sentence word by word